OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -23,6 +23,9 @@ public class CapturesInfo : ITraitInfo
 		[Desc("Types of actors that it can capture, as long as the type also exists in the Capturable Type: trait.")]
 		public readonly HashSet<string> CaptureTypes = new HashSet<string> { "building" };
 
+		[Desc("Triggers TransformOnCapture tagged by these types")]
+		public readonly HashSet<string> TransformTypes = new HashSet<string> { "husk" };
```

Comment:
The `TransformTypes` aren't used by these traits, so they should not be defined here.  This just introduces unwanted logical dependencies.